packages/datatree/frontend/sirius-components-datatree/src/DataTree.types.ts

```diff
@@ -25,7 +25,7 @@ export interface GQLDataTree {
 
 export interface GQLDataTreeNode {
   id: string;
-  parentId: string;
+  parentId: string | null;
```

Comment:
Fixed this incorrect typing from the data tree work.
This should have been `string | null` from the beginning: the GraphQL schema allows null value for parent id.